backend/src/test/resources/tqs/medex/frontend/listproducts.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Browse all available Products
+
+  Background:
+    Given I am logged in as the pharmacy owner
```

Comment:
Change to a unique message in Given so it isn't mistaken by other features and open other browsers just to log in, for example:
Given "I am the pharmacy owner on the list products page"

Reply:
Sure, I will do the appropriate changes.